wire-ios/Wire-iOS/Sources/UserInterface/Helpers/UIViewController+ErrorAlert.swift

```diff
@@ -64,18 +62,14 @@ extension UIViewController {
                 message = L10n.Localizable.Error.User.networkError
             case .emailIsAlreadyRegistered:
                 message = L10n.Localizable.Error.User.emailIsTaken
-            case .phoneNumberIsAlreadyRegistered:
-                message = L10n.Localizable.Error.User.phoneIsTaken
-            case .invalidPhoneNumberVerificationCode, .invalidEmailVerificationCode, .invalidActivationCode:
-                message = L10n.Localizable.Error.User.phoneCodeInvalid
+            case .invalidEmailVerificationCode, .invalidActivationCode: break
+               // message = L10n.Localizable.Error.User.phoneCodeInvalid
             case .registrationDidFailWithUnknownError:
                 message = L10n.Localizable.Error.User.registrationUnknownError
-            case .invalidPhoneNumber:
-                message = L10n.Localizable.Error.Phone.invalid
             case .invalidEmail:
                 message = L10n.Localizable.Error.Email.invalid
-            case .codeRequestIsAlreadyPending:
-                message = L10n.Localizable.Error.User.phoneCodeTooMany
+            case .codeRequestIsAlreadyPending: break
+                // message = L10n.Localizable.Error.User.phoneCodeTooMany
```

Comment:
Shall we use different strings instead of `break` here and a few lines above?

Reply:
Yes! Or try to re-introduce those two but with different names and content for the string.